USER_GUIDE.md

```diff
@@ -53,6 +53,24 @@ If `comments` is given in the `features` list then this enables all commenting f
 
 > Note: All commands can be given with a prefix of either `Derek <command>` or `/<command>`.
 
+### Feature: `messages`
```

Comment:
Is the feature named `message` or `messages`?  It makes sense for the yaml section to be called `messages` but the command the user types is `message`, which also makes sense.  The question is about consistency in terms the user being able to locate the detail for a command.

Arguably, it's an addition to the comments feature, rather than a feature in itself.

Reply:
The feature is message, moved the explanation in the bottom line of `USER_GUIDE.md` also extended the explanation to show how to trigger the message and swapped the toggle to `custom_messages` rather than `messages` to differentiate between the toggle and command.